cypress/e2e/shared/explorer.test.ts

```diff
@@ -1167,31 +1169,6 @@ describe('DataExplorer', () => {
       cy.get('.autorefresh-dropdown--pause').click()
       makeGraphSnapshot().shouldBeSameAs(snapshot, false)
     })
-
-    // skip until the auto-refresh feature is added back
-    it.skip('auto refresh', () => {
```

Comment:
Not too sure about removing this. Autorefresh is enabled in `oss`, maybe we should turn it on for that environment by moving this test there with a TODO to move it here?

Reply:
There were new auto-refresh e2e tests added: https://github.com/influxdata/ui/pull/1304 . So I don't think this skipped test is relevant anymore

Reply:
Those are for the dashboard page where the new feature is enabled, but the old feature is enabled in OSS.

Reply:
Oh I see. Sure, I can move the test to the oss/explorer.test.ts